### PR TITLE
Replaced non-Mozilla product Fluent terms w/ regular string in `/browsers/compare/` update

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
In a previous PR where we removed mentions of IE and updated Fluent string IDs, I kept the Fluent terms of non-Mozilla products in the new versions of the Fluent string IDs. Flod [flagged them in this PR](https://github.com/mozilla-l10n/www-l10n/pull/375). I managed to now replace the Fluent terms with strings.

Initial PR where these string IDs were created: https://github.com/mozilla/bedrock/pull/14167

i.e.
`string-about-safari = Comparing { -brand-name-safari }` -->  `string-about-safari = Comparing Safari`

## Issue / Bugzilla link

https://github.com/mozilla-l10n/www-l10n/pull/375

## Testing

- [ ] http://localhost:8000/en-US/firefox/browsers/compare/
- [ ] http://localhost:8000/en-US/firefox/more

### CODE REVIEW

**Comment by reemhamz** [comment]
@flodolo let me know if this is good to go

File: l10n/en/firefox/browsers/compare/index.ftl [hunk 1]
```diff
@@ -30,23 +30,23 @@ compare-index-and-the-winner-is = And the winner is…
 # Obsolete string
 compare-index-find-out-how-firefox-chrome = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera }, { -brand-name-brave } and { -brand-name-ie } differ in terms of privacy, utility and portability.
 
-compare-index-find-out-how-firefox-chrome-v2 = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-brave } differ in terms of privacy, utility and portability.
+compare-index-find-out-how-firefox-chrome-v2 = Find out how { -brand-name-firefox }, Chome, Edge, Safari, Opera and Brave differ in terms of privacy, utility and portability.
```
Comment (flodolo) [inline l10n/en/firefox/browsers/compare/index.ftl:33]:
```suggestion
compare-index-find-out-how-firefox-chrome-v2 = Find out how { -brand-name-firefox }, Chrome, Edge, Safari, Opera and Brave differ in terms of privacy, utility and portability.
```

File: l10n/en/firefox/browsers/compare/index.ftl [hunk 2]
```diff
@@ -30,23 +30,23 @@ compare-index-and-the-winner-is = And the winner is…
...
+compare-index-the-first-thing-to-point-out-v2 = The first thing to point out about portability is that not all browsers run on all operating systems. While { -brand-name-firefox }, Chrome, Edge, Brave and Opera work on all major systems and are easy to install, Safari only works on Microsoft and Apple’s own systems. The mobile version of Safari is pre-installed on Apple’s mobile devices, and most Android devices come with a pre-installed browser modified by the manufacturer for the device. { -brand-name-firefox }, Chrome, Brave, Edge and Opera can easily be installed and even used side by side.
```
Comment (flodolo) [inline l10n/en/firefox/browsers/compare/index.ftl:49]: Unrelated: Safari hasn't been available on Windows in decades? 🤔 
https://en.wikipedia.org/wiki/Safari_(web_browser)#Windows

**Review by flodolo — APPROVED** [review]
Thanks, that's exactly what I was asking for 👍🏼 

There is one typo, and one open question about Safari and Windows.

File: l10n/en/firefox/browsers/compare/index.ftl (hunk 2 above)
Comment (reemhamz) [reply l10n/en/firefox/browsers/compare/index.ftl]: Wow thanks for flagging!
I changed the string to `Safari only works on Apple’s own systems`

**Review by craigcook — APPROVED** [review]

**Pull request merged**